src/components/reusable/modal/modal.ts

```diff
@@ -119,6 +119,7 @@ export class Modal extends LitElement {
         <form method="dialog">
           <button
             class="close"
+            aria-label="close"
```

Comment:
1. The text "close" must be customizable for localization purposes.
2. It would be helpful to include this in the `title` attribute also.

Reply:
Addressed.